backend/tests/test_upload_reservation_routes.py

```
@@ -0,0 +1,151 @@
+        "surname": "Doe",
+        "birthday": "1990-01-01",
+        "street": "123 Main St",
+        "city": "Test City",
+        "province": "Test Province",
+        "cap": "12345",
+        "telephone": "1234567890",
+        "document_type": "ID",
+        "document_number": "ABC123456",
+        "cf": "JHNDOE90A01X123Y",
+        "frontImage": (
+            open("tests/test_images/front.jpeg", 'rb'),
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)